.github/workflows/ci.yml

```
@@ -16,11 +16,11 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: '1.14.9'
+          go-version: '1.17.5'
```

As Go 1.17.6 has been released in the meantime, we might as well upgrade to it :)
```suggestion
          go-version: '1.17.6'
```